### DISCUSSION TITLE
希望有人告诉

### DISCUSSION THREAD

**Discussion by Phaethon10031** [share121/inter-knot#1697] [其他]
绳网内怎么提升等级啊

**Comment by lllliufeng** [comment]
每有一个repositories升一级

**Comment by ArcSurge** [comment]
在GitHub上多fork几个仓库

**Reply by Fintbo** [reply]
等等，**你为什么有等级?**<img src="https://bbs-static.miyoushe.com/static/2024/06/19/a751ee1e27d4253ea0c233c905f36085_3464665030406935104.png" alt="托帕 眺望" width="75" >

**Reply by ArcSurge** [reply]
这个根据Github上的公开仓库数统计的，你多fork几个仓库就有了，不过确实没什么意义
<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >

**Comment by 00zzxx** [comment]
多开几个库就可以升级了，但是也没什么意义吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
